lib/core/src/display/rendering/htmlscene.rs

```diff
@@ -2,12 +2,39 @@ use crate::prelude::*;
 
 use super::HTMLObject;
 use super::Scene;
-use crate::data::opt_vec::OptVec;
+use crate::data::opt_vec::*;
```

Comment:
Updated it to
```
use crate::data::opt_vec::OptVec;
use crate::data::opt_vec::Iter;
use crate::data::opt_vec::IterMut;
```

Just a heads up. I started to use wildcards on the camera-system branch like this:

In `rendering/html/mod.rs`:
```rust
mod htmlscene;
pub use htmlscene::*;
```

This will make all the public API in `htmlscene` available in `basegl::rendering::html`, so you can use them as:

```rust
basegl::rendering::html::HTMLScene;
```

instead of

```rust
basegl::rendering::html::htmlscene::HTMLScene;
```